### PR TITLE
Added an android book and a Python playground

### PR DESCRIPTION
**Pull request by Bhaskar-maity**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Description 
Added an android book and a Python playground.
### Why is this valuable (or not)?
These websites are genuinely free.
### How do we know it's really free?
It shows ads and popular websites.
### For book lists, is it a book? For course lists, is it a course? etc.
1. android Book: javatpoint
2. python playground: Repl.it
### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-playgrounds.md [hunk 1]
```diff
@@ -153,6 +153,7 @@
 * [Codepad](http://codepad.org)
 * [Python Trinket](https://trinket.io/python)
 * [Python.org Shell](https://www.python.org/shell)
+* [Repl.it playground](https://repl.it/repls/NiceDualPoint#main.py)
```
Comment (eshellman) [inline free-programming-playgrounds.md:156]: don't invent titles. This one titles itself "repl.it - NiceDualPoint"

File: free-programming-books.md [hunk 2]
```diff
@@ -905,6 +905,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Coreservlets Android Programming Tutorial](http://www.coreservlets.com/android-tutorial/)
 * [Expert Android and Eclipse development knowledge](http://www.vogella.com/tutorials/android.html)
 * [Google Android Developer Training](https://developer.android.com/guide)
+* [Javatpoint Android Tutorial](https://www.javatpoint.com/android-tutorial)
```
Comment (eshellman) [inline free-programming-books.md:908]: belongs in tutorials

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by Bhaskar-maity** [comment]
I resolved it

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**